utilities/rpp-unittests/HIP_NEW/Tensor_hip_pln3.cpp

```diff
@@ -151,6 +151,9 @@ int main(int argc, char **argv)
     case 31:
         strcpy(funcName, "color_cast");
         break;
+    case 35:
+        strcpy(funcName, "glitch");
```

Comment:
I think glitch case is not added in HIP backend PKD3 unit test file and performance test files
Could not find them in the file changes
Please check and add them if not already added